README.md

```diff
@@ -46,59 +46,62 @@ return [
 
 ## Usage
 
-The `@DeprecatedRoute` annotation extends `Symfony\Component\Routing\Annotation\Route`, and lets you mark an endpoint as deprecated and/or expired (and optionally make the endpoint inaccessible) by exposing these options: 
-- `since` is a date value `(dd-mm-yyyy)` that defines the moment in which a route becomes deprecated. If the current date is equal or greater than the value of since, the header `Deprecation` will be set on the response, like so:
+You can deprecate a route in any route definition (annotation, yaml, xml, php, what have you) by passing three values to the `defaults` option:
+ 
+- `_deprecated_since` is a `string ("dd-mm-yyyy")` that defines the moment in which a route becomes deprecated. If the current date is equal or greater than the value of since, the header `Deprecation` will be set on the response, like so:
  `Deprecation: date="Wed, 01 Jan 2020 00:00:00 GMT"`.
  
-- `until` is a date value `(dd-mm-yyyy)` that defines the moment in which a route becomes expired. If the current date is equal or greater than the value of until, the header `Sunset` will be set on the response, like so:
+- `_deprecated_until` is a `string ("dd-mm-yyyy")` that defines the moment in which a route becomes expired. If the current date is equal or greater than the value of until, the header `Sunset` will be set on the response, like so:
```

Comment:
that's not true. The `Sunset` header is always set when the route is deprecated an this value is set.

Reply:
> that's not true. The `Sunset` header is always set when the route is deprecated an this value is set.

Could you add a bug issue?

Reply:
https://github.com/halloverden/symfony-route-deprecation-bundle/issues/11